src/microbit/pxt_hermegaasa/hermegaasa.md

```diff
@@ -0,0 +1,249 @@
+
+- [ ] Finn `når radio mottar receivedString` fra *Radio*.
+
+- [ ] Vis teksten `receivedString` på skjermen.
+
+- [ ] Opprett en ny variabel med navn `Press` og sett denne til `receivedString`. Da har vi lagret dette til å sjekke om vi har klart å herme etter spillederen.
+
+- [ ] Dersom du har gjort alt rett så vil koden din se slik ut:
+
+  ![](player_receive.png)
+
+# Steg 8: Når spiller mottar melding {.activity}
+
+*Siste steg er å herme etter spilleder og sende beskjed i retur.*
+
+## Sjekkliste {.check}
+
+- [ ] Vi starter med å herme etter spilleder dersom meldingen var `A`. Først må vi finne blokken som heter `når knapp A trykkes`. Den ligger under *Inndata*.
+
+- [ ] Så henter vi `hvis-blokken` fra *Logikk* og setter den sammen med `erlik-blokken` (`=`) fra samme kategori. Her skal du sjekke om `Press` er lik `A`.
+
+- [ ] Dersom `Press` er lik `A` så skal vi sende spillernummeret vårt tilbake til spilleder. Legg til blokken `radio send tall` inni `hvis-blokken` og koble den til `Player`. Legg også til blokken `tøm skjermen` fra *Basis*.
+
+- [ ] Utvid `hvis-blokken` til `hvis - ellers`.
+
+- [ ] `Press` ikke er lik `A` så skal man vise et surt fjes. Du finner blokken `vis ikon` under *Basis*. Velg et passende ikon.
+
+- [ ] Legg også til `pause` i `500` ms. Til slutt kan du finne blokken `vis tekst` og legge variabelen `Press` inni den, for å vise hva du egentlig burde gjort.
+
+- [ ] Dersom du har gjort alt rett så vil koden din se slik ut:
+
+  ![](player_press_a.png)
+
+- [ ] Gjør tilsvarende for `B`, `C`, `<` og `>`.
+
+- [ ] Dersom du har gjort alt rett så vil koden din se slik ut:
+
+  ![](player_press_all.png)
+
+# Steg 5: Overføre programmet til spillerne {.activity}
+
+*Overfør programmet til hver av spillerne sine micro:biter. Husk å endre programmet med spillernummer mellom hver gang og last ned på nytt.*
+
+## Sjekkliste {.check}
+
+- [ ] Gi navnet `spiller`til programmet ditt. Dette gjør du ved å endre teksten helt nede på midten på skjermen din.
+
+- [ ] Koble den enen micro:biten din til datamaskinen med en USB-kabel. Denne micro:biten blir spillederen i spillet.
+
+- [ ] Klikk deretter på knappen `Last ned` nede til venstre på skjermen. Det lastes nå ned en fil som heter `spiller.hex` til datamaskinen din.
+
+- [ ] Flytte denne filen over til MICROBIT-disken på maskinen din.
+
+- [ ] Endre spillernummer i programmet og gjør samme prosess for hver av spillerne som skal være med.
+
+### OBS! {.protip}
+
+Husk å endre hvilket nummer hver spiller er mellom hver nye microbit du kobler til og laster ned programmet til. Du må laste ned programmet på nytt for hver av spillerne med de endringene du har gjort.
+
+## Utfordringer {.challenge}
+
+- [ ] Legge til flere elementer som spillerne skal reagere på. For eksempel dersom micro:biten ristes.
+
+- [ ] Endre koden til spilleder slik at Micro:biten automatisk velger helt tilfeldig hva spillerne skal herme etter.
+
+- [ ] Endre koden slik at det er mulig å jukse i spillet.
```

Comment:
Ref kommentar over. Serveren muliggjør juks. Men her snakker vel du kun om å skrive en annen spillklient som jukser? Synes for øvrig det er OK slik det står nå.